internal/commands/app/create.go

```diff
@@ -187,8 +187,12 @@ func (cmd *CommandCreate) Handler(profile *user.Profile, ui terminal.UI, clients
 	}
 
 	var appLocal local.App
-
-	if appRemote.GroupID == "" && appRemote.ID == "" {
+	if cmd.inputs.Template != "" {
```

Comment:
i'm actually fine with it laid out this way as its reads nice/clear/direct, but I just want to call out `createFromTemplate` does a lot of the same things as the `else` block here does.  at the cost of making the `else if` here a bit more complex (`if appRemote.GroupID == "" && appRemote.ID == "" && cmd.inputs.Template == ""`, you could then leave the `else` block as is and add the `downloadAndWriteClient` usage inside a `if cmd.inputs.Template != ""` block

the only reason I suggest this is should the process of setting `appLocal` ever change (export => write => load) then we'll need to remember two spots in this command to update.  not a huge deal, i could just see us missing that at some point down the line.

[tangent] I want to wait and see how templates evolve before fully going down this path, but it seems to me that getting the `frontendDir`/`backendDir` management (with respect to `appLocal.RootDir`) to be owned/controlled by the `local.App` itself would make writing these commands a bit easier.  If we added something like `TemplateID` to the `local.App` struct, I think we could then remove a lot of the directory/path management we do for templates in `app create`/`push`/`pull`/`app init`...

Reply:
yeah it does do a lot of the same thing. I found it made more sense with the least amount of refactoring to duplicate some of that code so all template logic was in the same place.

I'm open to doing a larger refactor later, but since we are pushing to release template starter apps I'm going to leave as is.